conf/benchmark/mlss_conf.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<compara_db division="benchmark">
+
+  <collections>
+
+    <collection name="dev_test">
```

Comment:
If you plan running gene trees pipelines maybe "default" might make your life easier in terms of configuration. But who doesn't like an additional challenge? :sunglasses:

Reply:
Ah, I didn't think that'd be an issue. Would it require much fiddling around beside e.g. setting `'collection' => 'dev_test'` in `ProteinTrees_conf.pm`?
I was planning to use `default` for a proper size dataset, e.g. the one that will be used for benchmarking.
I could name this one `default` to spare myself some trouble or keep it as it is for the learning purposes O:)

Reply:
Keep it! Keep it! :smiling_imp: